### PR TITLE
Update Copyright headers to use OpenCensus Authors. Add AUTHORS file.

### PR DESCRIPTION
**Pull request by chingor13**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -186,7 +186,7 @@ file or class name and description of purpose be included on the
 same "printed page" as the copyright notice for easier
 identification within third-party archives.
 
-Copyright {yyyy} {name of copyright owner}
+Copyright 2017 OpenCensus Authors
```
Comment (tmatsuo) [inline LICENSE]: You should keep this as it is. Same for other `LICENSE` files

Comment (bogdandrutu) [inline LICENSE]: Correct. Please do not change this file.

File: ext/LICENSE [hunk 2]
```diff
@@ -186,7 +186,7 @@ file or class name and description of purpose be included on the
 same "printed page" as the copyright notice for easier
 identification within third-party archives.
 
-Copyright [yyyy] [name of copyright owner]
+Copyright 2017 OpenCensus Authors
```
Comment (bogdandrutu) [inline ext/LICENSE]: Same here.

File: LICENSE (hunk 1 above)
Comment (chingor13) [inline LICENSE]: Ok, will change back.

File: ext/LICENSE (hunk 2 above)
Comment (chingor13) [inline ext/LICENSE]: Ok, will change back.

**Comment by chingor13** [comment]
@bogdandrutu I was going to squash/merge, but I can squash first.

**Pull request merged**